src/addScore.js

```diff
@@ -0,0 +1,31 @@
+const submitScore = () => {
+  const scoreForm = document.getElementById('scoreForm');
+  scoreForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const userNameInput = document.getElementById('userNameInput');
+    const scoreInput = document.getElementById('scoreInput');
+
```

Comment:
- [OPTIONAL] Your app works as expected. :+1: I'll suggest clearing the input after submitting a new score. It will give a better user experience. :smile: :+1:

Reply:
Thank you for the review and feedback @V-Blaze , much appreciated.

Thats a brilliant idea, I will certainly implement your recommendation.

Thank you😄 👍